packages/config/src/layer2s/templates/polygonCDKStack.ts

```diff
@@ -0,0 +1,577 @@
+              'https://etherscan.io/address/0x5132A183E9F3CB7C848b0AAC5Ae0c4f0491B7aB2',
+            ],
+          },
+        ],
+      },
+      destinationToken: RISK_VIEW.NATIVE_AND_CANONICAL(),
+      validatedBy: RISK_VIEW.VALIDATED_BY_ETHEREUM,
+    }),
+    stage:
+      daProvider !== undefined
+        ? { stage: 'NotApplicable' }
+        : getStage(
+            {
+              stage0: {
+                callsItselfRollup: true,
+                stateRootsPostedToL1: true,
+                dataAvailabilityOnL1: true,
+                rollupNodeSourceAvailable: true,
+              },
+              stage1: {
+                stateVerificationOnL1: true,
+                fraudProofSystemAtLeast5Outsiders: null,
+                usersHave7DaysToExit: false,
+                usersCanExitWithoutCooperation: false,
+                securityCouncilProperlySetUp: [
+                  false,
+                  'Security Council members are not publicly known.',
+                ],
+              },
+              stage2: {
+                proofSystemOverriddenOnlyInCaseOfABug: false,
+                fraudProofSystemIsPermissionless: null,
+                delayWith30DExitWindow: false,
+              },
+            },
+            {
+              rollupNodeLink: 'https://github.com/0xPolygonHermez/zkevm-node',
+            },
+          ),
+    technology: {
+      newCryptography: templateVars.nonTemplateTechnology?.newCryptography,
+      stateCorrectness: templateVars.nonTemplateTechnology
+        ?.stateCorrectness ?? {
+        ...STATE_CORRECTNESS.VALIDITY_PROOFS,
+        references: [
+          {
+            text: 'PolygonRollupManager.sol - Etherscan source code, _verifyAndRewardBatches function',
+            href: 'https://etherscan.io/address/0x5132A183E9F3CB7C848b0AAC5Ae0c4f0491B7aB2',
+          },
+        ],
+      },
+      dataAvailability:
+        templateVars.nonTemplateTechnology?.dataAvailability ??
+        technologyDA(daProvider),
+      operator: templateVars.nonTemplateTechnology?.operator ?? {
+        name: 'The system has a centralized sequencer',
+        description:
+          'Only a trusted sequencer is allowed to submit transaction batches. A mechanism for users to submit their own batches is currently disabled.',
+        risks: [
+          FRONTRUNNING_RISK,
+          {
+            category: 'Funds can be frozen if',
+            text: 'the sequencer refuses to include an exit transaction.',
+            isCritical: true,
+          },
+        ],
+        references: [],
+      },
+      forceTransactions: templateVars.nonTemplateTechnology
+        ?.forceTransactions ?? {
+        ...FORCE_TRANSACTIONS.SEQUENCER_NO_MECHANISM,
+        description:
+          'The mechanism for allowing users to submit their own transactions is currently disabled.',
+      },
+      exitMechanisms: templateVars.nonTemplateTechnology?.exitMechanisms ?? [],
```

Comment:
rather than empty this can be default as they share the same bridge / claimAsset function:
      {
        ...EXITS.REGULAR('zk', 'merkle proof'),
        references: [
          {
            text: 'PolygonZkEvmBridgeV2.sol - Etherscan source code, claimAsset function',
            href: 'https://etherscan.io/address/0x0feb850b183c57534b56b7d56520133c8f9bdb65',
          },
        ],
      },